lib/protocol/bep_extensions.go

```diff
@@ -117,13 +117,34 @@ func (f FileInfo) WinsConflict(other FileInfo) bool {
 	return f.Version.Compare(other.Version) == ConcurrentGreater
 }
 
+func (f FileInfo) NonExistent() bool {
```

Comment:
I'd prefer this to be named something else for clarity, maybe `IsZero` or `IsEmpty`. Saying `NonExistent` feels like it might return false if the FileInfo represents a file that doesn't exist on disk or something like that.

Reply:
Also, why not just compare it to protocol.FileInfo{}? We could add an Equal method can compare with that?

Reply:
An `Equals()` would be fairly involved what with all the slices and stuff, and we don't actually need one so far. I think the *implementation* here is fine, but the naming is misleading.